### ISSUE TITLE
Unused parameters, module version complete?

### ISSUE THREAD

**Issue by claflico** [joshbeard/puppet-crowd#6]
Sorry for asking but I am using 1.0.4 from repoforge and am curious if the released version is complete. There are some parameters (such as dbpassword) that don't seem to be used anywhere past init.pp. After install I still had to go through the wizard and specify the postgresql connection info and enter username/password which I would think it would get from the module.

Thanks.

**Comment by joshbeard** [comment]
Good catch.  These are leftover from the upstream module this was forked from.  I can't recall if Crowd can have the database stuff setup without going through the wizard or not.  I know there's some variance among the Atlassian suite for this sort of thing.

**Comment by claflico** [comment]
Yeah, unfortunately I'm discovering that as I'm preparing to replace our snowflake installs with puppet-managed ones.....there's little consistency among the atlassian products.
